pkg/controller/utils/status.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"github.com/go-logr/logr"
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// SetAccountStatus sets the condition and state of an account
+func SetAccountStatus(reqLogger logr.Logger, awsAccount *awsv1alpha1.Account, message string, ctype awsv1alpha1.AccountConditionType, state string) {
```

Comment:
I get why you moved this, but IMHO it's more appropriate to keep it in the account package to highlight the fact that the account package ought to own changes to Account CRs.